### PR TITLE
Only allow single slash at start of scriptUrl.

### PR DESCRIPTION
**Pull request by guidobouman**
Fixes default `publicPath ` colliding with absolute `filename`.

---

Okay, a second PR. But we might need to test this a lot more.

When using the default public path & a static file URL, you get two slashes in front of the URL. This causes Chrome to refuse installing the serviceworker because of failing CORS checks. The CORS check fails because two slashes indicate a protocol-agnostic URL (including domain.) Where one slash would indicate an absolute URL-path.

This problem surfaced because my previous PR.

---

This PR might not really be necessary as you can set `publicPath ` to `''` to fix this. That would make sense actually... But might need proper documentation, before other people trip on this.

What are your thoughts?

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ export default class ServiceWorkerPlugin {
         // Hijack the original module
         if (result.resource === runtimePath) {
           const data = {
-            scriptURL: this.options.publicPath + this.options.filename,
+            scriptURL: (this.options.publicPath + this.options.filename).replace(/^\/{2,}/, '/'),
```
Comment (woutervanvliet) [inline src/index.js]: Merge conflicts.

And, isn't this problem better solved by using `path.normalize`?

Or, even better, by using `path.join` instead if string concat.

Comment (guidobouman) [reply src/index.js]: Great idea!

**Comment by guidobouman** [comment]
Great feedback! Let me look at the testcase.

**Comment by woutervanvliet** [comment]
I've implemented a test case for your change, and submitted a new pull request including the test case. See: #57

**Pull request closed without merging**